src/System.Numerics.Vectors/tests/GenericVectorTests.cs

```diff
@@ -469,10 +469,10 @@ static GenericVectorTests()
             Vector<T> v1 = new Vector<T>(values1);
             int hash = v1.GetHashCode();
 
-            int expected = 0;
+            int expected = 1;
             for (int g = 0; g < Vector<T>.Count; g++)
             {
-                expected = (((expected << 5) + expected) ^ v1[g].GetHashCode());
+                expected = 31 * expected + v1[g].GetHashCode();
```

Comment:
I Agree, whole file GenericVectorTests.tt is a T4 template. There are a lot of generated code, I decided not to toch it.